tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py27, py37
+isolated_build = True
+
+[testenv]
+deps =
+  cookiecutter
+  flake8
+  pytest
+  tomlkit
```

Comment:
TIL about tomlkit... I've been using vanilla `toml` in briefcase. Based on [this recommendation](https://twitter.com/uranusjr/status/1265812453599764481) from a PyPA member, I guess we should stick to toml, as we're only using the toml *parsing* capability, not the writing features.

Reply:
Oh, interesting! Thanks for sharing. I'd actually only recently come across tomlkit, but that was a context where I needed to write to file as well.